src/CSS/Create.scss

```diff
@@ -23,6 +23,15 @@ h2{
   align-items: center;
 }
 
+#cancelButton {
+  color:#d04c55;
+  width: 5em;
+  height: 2.6em;
```

Comment:
don't fix height nor width on a button: if your text content increase (e.g. using font-size), the design will break. prefer padding (usually with more padding on left and right):

    padding: .5rem 1rem;